hw/ip/aes/README.md

```diff
@@ -60,4 +60,4 @@ Other cipher modes might be added in future versions.
 The AES unit is attached to the chip interconnect bus as a peripheral module.
 Communication with the processor happens through a set of control and status registers (CSRs).
 This includes input/output data and key, as well as status and control information.
-Future versions of the AES unit might include a separate interface through which a possible system key manager can provide the key without exposing it to the processor or other hosts attached to the system bus interconnect.
+In addition, the AES unit includes a separate interface through which [Key Manager](../keymgr/README.md) can provide the key without exposing it to the processor or other hosts attached to the chip interconnect bus.
```

Comment:
I'm not a native speaker, but I would use an article before "Key Manager".
Anyways, the meaning is now clear.

Reply:
I believe the article is not required here as there only one keymgr in the system / OpenTitan context :-)